testing/zerotier-one/zerotier-one.initd

```diff
@@ -1,11 +1,14 @@
 #!/sbin/openrc-run
 
-depend() {
-    provide net
-    use network
-    before firewall netmount
-}
-
+name=zerotier-one
 command="/usr/sbin/zerotier-one"
+command_args="$zerotier_opts"
 command_background="yes"
-pidfile="/run/$RC_SVCNAME.pid"
+
+start_stop_daemon_args=""
+pidfile="/var/lib/zerotier-one/zerotier-one.pid"
```

Comment:
PID files should be in `/run` (typically `/run/$RC_SVCNAME.pid`), definitely not in `/var/lib`!